source/support.tex

```diff
@@ -2236,7 +2237,7 @@
 the program shall also define
 the corresponding version without the \tcode{size} parameter.
 \begin{note}
-The default behavior below might change in the future, which will require
+The default behavior below can change in the future, which will require
 replacing both deallocation functions when replacing the allocation function.
```

Comment:
I think this has lost the intent, which is that we've identified this as a desirable change. How about:
```suggestion
It is anticipated that the default behavior will change in the future and will instead require
replacing both deallocation functions when replacing the allocation function.
```